clippy_lints/src/str_to_string.rs

```diff
@@ -0,0 +1,57 @@
+use crate::utils::span_lint_and_help;
+
+use if_chain::if_chain;
+use rustc_hir as hir;
+use rustc_hir::*;
```

Comment:
NIT (better to avoid wildcard imports).

```suggestion
use crate::utils::span_lint_and_help;
use if_chain::if_chain;
use rustc_hir as hir;
```